lib/renderer/api/remote.js

```diff
@@ -7,6 +7,10 @@ const { CallbacksRegistry } = require('@electron/internal/renderer/remote/callba
 const { isPromise, isSerializableObject } = require('@electron/internal/common/remote/type-utils')
 const { ipcRendererInternal } = require('@electron/internal/renderer/ipc-renderer-internal')
 
+const {
+  Promise
+} = capturedGlobals
```

Comment:
can we have webpack automatically inject this instead of modifying every single JS file?

Reply:
Technically, yes.  We could make webpack provide `Promise`.  I did it this way to somewhat align with how primordials _look_.  I can do a Promise provide if that's the kind of magic we want to do 👍

Reply:
ah ok, I'm not familiar with node's primordials. I'm fine with this style if it's matching node!